public/app/plugins/datasource/prometheus/types.ts

```diff
@@ -10,13 +10,16 @@ export interface PromQuery extends DataQuery {
   hinting?: boolean;
   interval?: string;
   intervalFactor?: number;
+  stepMode?: StepType;
```

Comment:
We call the same thing `stepMode`, `StepType`, `STEP_OPTION`. We should unify the naming to `stepMode`.

Reply:
Yes you're right it should be uniform!